packages/preview/tongji-undergrad-thesis-typst/0.1.0/typst.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "tongji-undergrad-thesis-typst"
```

Comment:
This name doesn't follow the guidelines for two reasons:
- it should not include "typst".
- it should also not be the most canonical/obvious name for what it does. Adding an adjective in front of `tongji-undergrad-thesis` to make the name more unique and easily identifiable would be enough.